packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapMarkerController.m

```diff
@@ -127,14 +139,14 @@ - (void)interpretMarkerOptions:(NSDictionary *)data
   if (rotation) {
     [self setRotation:[rotation doubleValue]];
   }
-  NSNumber *visible = FGMGetValueOrNilFromDict(data, @"visible");
-  if (visible) {
-    [self setVisible:[visible boolValue]];
-  }
   NSNumber *zIndex = FGMGetValueOrNilFromDict(data, @"zIndex");
   if (zIndex) {
     [self setZIndex:[zIndex intValue]];
   }
+  NSNumber *visible = FGMGetValueOrNilFromDict(data, @"visible");
+  if (visible) {
+    [self setVisible:[visible boolValue]];
+  }
```

Comment:
Moving this code doesn't seem necessary to your change. Please revert to minimise diffs.